### ISSUE TITLE
Spaces are not allowed between list and (

### ISSUE THREAD

**Issue by zhujinxuan** [bobthecow/psysh#549]
I am not sure it is a problem from PHP-parser or a problem of psysh.  The following code gives an error:

```
>>> list ($a, $b) = [1,2];
Unknown target: ($a, $b) = [1,2]
```

**Comment by bobthecow** [comment]
Ha! Thanks for reporting this.

There's an `ls` command in psysh, and it has the alias `list`. Until pretty recently, it wasn't ambiguous because it could only take a static argument, e.g. `list $a`.

But commands like `ls` now accept expressions as well (which is awesome) so now it's ambiguous when you type `list ($a, $b) = [1, 2]` whether you mean `ls (result of expression)` or the PHP keyword. And I never noticed because I never use a space after `list` 😄 

I'll remove the `list` alias. In the meantime, you can always get the non-command version by prefixing with `;`…

<img width="762" alt="screen shot 2019-02-18 at 8 37 52 am" src="https://user-images.githubusercontent.com/53660/52964955-7e437700-3358-11e9-9c24-f44e7a1f1de1.png">

**Comment by bobthecow** [comment]
I pushed a fix in beef754, and it will go out in the next stable release.

**Comment by zhujinxuan** [comment]
Great.  I will closed this issue because it is fixed!
